packages/core/src/input/Pointer.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Pointer.
+ */
+export class Pointer {
+  /**
+   * Constructor a Pointer.
+   * @param pointerId - The pageX of the pointer
+   * @param pageX - The pageY of the pointer
+   * @param pageY - The pointerId of the pointer
+   * @param indexInList - The index of the pointer in pointerList
+   */
+  constructor(public pointerId: number, public pageX: number, public pageY: number, public indexInList: number) {
```

Comment:
Desogn is in https://github.com/oasis-engine/engine/issues/506 , indexInList is internal, shouldn't public

Reply:
Done.